DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,7 @@ private function addSpamAlertsSection(ArrayNodeDefinition $node)
                         ->scalarNode(AzineMailgunWebhooksExtension::TICKET_SUBJECT)->defaultValue('IP on spam-list, please fix.')->info('Mailgun HelpDesk ticket subject')->end()
                         ->scalarNode(AzineMailgunWebhooksExtension::TICKET_MESSAGE)->defaultValue('It looks like my ip is on a spam-list. Please, assign a clean IP to my domain.')->info('Mailgun HelpDesk ticket subject')->end()
                         ->scalarNode(AzineMailgunWebhooksExtension::ALERTS_RECIPIENT_EMAIL)->defaultValue('')->info('Admin E-Mail to send notification about spam complaints')->end()
+                        ->scalarNode(AzineMailgunWebhooksExtension::SENT_EMAIL_ADMIN_OVERVIEW_ROUTE)->defaultValue('azine_admin_email_dashboard')->info('The route to admin email dashboard from AzineEmailBundle')->end()
```

Comment:
Replace the info-text with "Route name to use to generate a WebView link in the Mailgun Event List."